### PR TITLE
employed the modified python sdk

### PR DESCRIPTION
**Pull request by rohshar**

### CODE REVIEW

File: middleware.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import requests
 from celery import Celery
 from flask import Flask, request, session, redirect, url_for, render_template, flash, Response, jsonify
+from carto import *
```
Comment (danicarrion) [inline middleware.py]: call me picky, but please don't use \* but write the actual names you need

File: middleware.py [hunk 2]
```diff
@@ -148,10 +146,8 @@ def delete_token(token):
 
     # Call CartoDB's Maps API to remove the token from the named map
     # This will also invalidate the cache for the tiles on the CDN with this token
-    requests.put(os.path.join(config.get('cartodb', 'maps_endpoint'), "named", config.get('map', 'name')),
-                 data=json.dumps(named_map),
-                 params={"api_key": config.get('cartodb', 'api_key')},
-                 headers={'content-type': 'application/json'})
+    auth_client = APIKeyAuthClient(config.get('cartodb', 'api_key'), config.get('cartodb', 'username'))
+    auth_client.send(os.path.join(config.get('cartodb', 'maps_endpoint'), "named", config.get('map', 'name')), http_method="PUT", http_headers={'content-type': 'application/json'}, body=json.dumps(named_map), params={"api_key": config.get('cartodb', 'api_key')})
```
Comment (danicarrion) [inline middleware.py:150]: why aren't you using a NamedMap object for this?

File: middleware.py [hunk 3]
```diff
@@ -162,10 +158,9 @@ def map_items():
     named_map["auth"]["valid_tokens"].append(new_token)
 
     # Call CartoDB's Maps API to add the token to the named map
-    requests.put(os.path.join(config.get('cartodb', 'maps_endpoint'), "named", map_name),
-                 data=json.dumps(named_map),
-                 params={"api_key": config.get('cartodb', 'api_key')},
-                 headers={'content-type': 'application/json'})
+    auth_client = APIKeyAuthClient(config.get('cartodb', 'api_key'), config.get('cartodb', 'username'))
+    auth_client.send(os.path.join(config.get('cartodb', 'maps_endpoint'), "named", map_name), http_method="PUT", http_headers={'content-type': 'application/json'}, body=json.dumps(named_map), params={"api_key": config.get('cartodb', 'api_key')})
```
Comment (danicarrion) [inline middleware.py:162]: same as above

**Comment by rohshar** [comment]
I wasn't able to used a namedMap object, this is because these two parts deal with tokens, and I can't find the logic for that in the maps API

**Pull request closed without merging**